### PR TITLE
Feature/quaternion conversion

### PR DESCRIPTION
**Pull request by aestene**

### CODE REVIEW

**Comment by aestene** [comment]
Awaiting bug fix for Euler angle order.

File: setup.cfg [hunk 1]
```diff
@@ -14,3 +14,7 @@ ignore_missing_imports = True
 
 [mypy-scipy.spatial.*]
 ignore_missing_imports = True
+
+[mypy-scipy.spatial.transform.rotation]
+ignore_missing_imports = True
+
```
Comment (Christdej) [inline setup.cfg:20]: Was this needed to make mypy happy? (I would believe the one above would allow all under scipy.spatial

**Review by Christdej — APPROVED** [review]

File: tests/convert/test_quaternion.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                to_="asset",
+            ),
+        ),
+    ],
+)
+def test_quaternion_to_euler(quaternion, expected):
+    euler_angles: Euler = quaternion_to_euler(
+        quaternion=quaternion, sequence="ZYX", degrees=False
+    )
+    # assert euler_angles == expected
```
[the remark this answers is not in the document]
Comment (aestene) [reply tests/convert/test_quaternion.py]: Removed.

File: setup.cfg (hunk 1 above)
Comment (aestene) [reply setup.cfg]: Good catch! I think I missed it in a rebase :D It has been removed now!

**Pull request merged**